capella_console_client/enumerations.py

```diff
@@ -123,3 +123,10 @@ class LocalTimeOption(str, BaseEnum):
     day = "day"
     night = "night"
     anytime = "anytime"
+
+
+class OwnershipOption(str, BaseEnum):
+    owned_by_organization = "ownedByOrganization"
```

Comment:
nice - please UPPERCASE and shorten the keys to something like

```python
class OwnershipOption(str, BaseEnum):
    ORG = "ownedByOrganization"
    PUBLIC = "publiclyAvailable"
    SHARED = "sharedWithOrganization"
    PURCHASABLE = "availableForPurchase"
```

https://docs.python.org/3/howto/enum.html

> ... because Enums are used to represent constants, and to help avoid issues with name clashes between mixin-class methods/attributes and enum names, we strongly recommend using UPPER_CASE names for members, and will be using that style in our examples.

Reply:
Will do! The other examples in this file weren't uppercase, is this because they are matching CAPI attributes?

Reply:
that's right